# Header for timestamp etc.
Header header

# Pedal connected?
bool connected

# Pedals pressed?
bool left_pressed
bool middle_pressed
bool right_pressed
